backend/scheduler/src/jobs/resolve-sports-markets.ts

```diff
@@ -0,0 +1,120 @@
+import { createSupabaseDirectClient } from 'shared/supabase/init'
+import { log } from 'shared/utils'
+import { Contract, CPMMMultiContract } from 'common/contract'
+import { HOUSE_LIQUIDITY_PROVIDER_ID } from 'common/antes'
+import { resolveMarketHelper } from 'shared/resolve-market-helpers'
+import { SportsGames } from 'common/sports-info'
+import { api } from 'web/lib/api/api'
+
+export async function resolveSportsMarkets() {
+  const pg = createSupabaseDirectClient()
+
+  try {
+    const data = await api('get-completed-sports-games', {})
+    const completedGames = (data.schedule || []) as SportsGames[]
+    log(`Processing ${completedGames.length} completed games.`)
+
+    const unresolvedContracts = await pg.map<Contract>(
```

Comment:
why not just filter the contracts where data->>'sportsEventId' is in your completed games id array?